ui/src/containers/KindFilter/__snapshots__/KindFilter.test.tsx.snap

```diff
@@ -19,8 +19,8 @@ exports[`KindFilter finds the filter component and matches the count 1`] = `
             </GridItem>
             <GridItem span={3}>
               <div className=\\"pf-l-grid__item pf-m-3-col\\">
-                <Button variant=\\"plain\\" aria-label=\\"Clear\\" onClick={[Function: res]}>
-                  <button onClick={[Function: res]} aria-disabled={false} aria-label=\\"Clear\\" className=\\"pf-c-button pf-m-plain\\" disabled={false} tabIndex={[undefined]} type=\\"button\\" data-ouia-component-type=\\"PF4/Button\\" data-ouia-safe={true} data-ouia-component-id={1}>
+                <Button variant=\\"plain\\" aria-label=\\"Clear\\" onClick={[Function: res] { _isMSTAction: true }}>
+                  <button onClick={[Function: res] { _isMSTAction: true }} aria-disabled={false} aria-label=\\"Clear\\" className=\\"pf-c-button pf-m-plain\\" disabled={false} tabIndex={[undefined]} type=\\"button\\" data-ouia-component-type=\\"PF4/Button\\" data-ouia-safe={true} data-ouia-component-id={1}>
```

Comment:
I was wondering why the snapshot of the filters are changed? :thinking:

Reply:
I think bcoz of Search and App component changes,let me check again

Reply:
these changes are happening bcoz of button's onclick function is a mst action

Reply:
Are these changes expected ? Because I think all components are independent of each other

Reply:
I think it might be because in filters we might be searching for a resource